src/main/scala/rocket/ALU.scala

```diff
@@ -27,6 +26,46 @@ object ALU
   def FN_SLTU = 14.U
   def FN_SGEU = 15.U
 
+  // from Zb
+  // Zba: UW is encoded here becuase it is DW_64
+  def FN_ADDUW    = 0.U
```

Comment:
Can you make these throw an exception if they are accessed? If `ALU` is selected instead of `ABLU`, then no code should try to access `ALU.FN_ADDUW` right?

Reply:
Used `???`